src/java/org/apache/cassandra/cql3/restrictions/StatementRestrictions.java

```diff
@@ -833,12 +833,22 @@ private void validateSecondaryIndexSelections()
      * @return <code>true</code> if all the primary key columns are restricted by an equality relation.
      */
     public boolean hasAllPKColumnsRestrictedByEqualities()
```

Comment:
```suggestion
    public boolean hasAllPrimaryKeyColumnsRestrictedByEqualities()
```
Might as well go for the full name w/ `hasAllPartitionKeyColumnsRestrictedByEqualities()` doing the same